### PR TITLE
Update template

### PR DESCRIPTION
**Pull request by rowanc1**
This makes is to it builds, but now with lots of typst errors/changes.

### CODE REVIEW

File: examples/example-myst/myst.yml [hunk 1]
```diff
@@ -17,18 +17,18 @@ project:
   bibliography: []
   exports:
     - format: typst
-      template: typst-thesis.typ
+      template: ../../thesis-template
```
Comment (rowanc1) [inline examples/example-myst/myst.yml:20]: This has to be the folder, not a file. It is picking up the folder with the template.yml in it.

File: examples/example-myst/myst.yml [hunk 2]
```diff
@@ -17,18 +17,18 @@ project:
...
       articles:
-      - file: examples/example-myst/00-Front-Matter/01-Declaration.md
-      - file: examples/example-myst/00-Front-Matter/02-Abstract.md
-      - file: examples/example-myst/00-Front-Matter/03-Preface.md
-      - file: examples/example-myst/00-Front-Matter/04-Acknowledgements.md
-      - file: examples/example-myst/00-Front-Matter/05-Dedication.md
-      - file: examples/example-myst/00-Front-Matter/06-Glossary.md
-      - file: examples/example-myst/01-Chapters/01-Introduction.md
-      - file: examples/example-myst/01-Chapters/02-Chapter.md
+        - file: 00-Front-Matter/01-Declaration.md
```
Comment (rowanc1) [inline examples/example-myst/myst.yml:24]: The myst project should be distinct, eventually you might spin the `template` folder off into a small stand-alone repo that can be shared and pointed to independently.

File: thesis-template/template.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ myst: v1
 kind: typst
 title: Thesis Template
 description: Typst-MyST Thesis Template
-template: typst-thesis.typ
+template: template.typ
```
Comment (rowanc1) [inline thesis-template/template.yml:5]: This was the main reason things were blank, the typst-thesis.typ builds a blank PDF always.

**Review by rowanc1 — COMMENTED** [review]
Moved some things around and got some errors that are now typst/styling...

File: examples/example-myst/myst.yml (hunk 1 above)
Comment (roaldarbol) [reply examples/example-myst/myst.yml:20]: Aaaaah, that makes sense! No idea that was the case!

**Pull request merged**